src/utils/index.js

```diff
@@ -1,5 +1,27 @@
+import React from 'react';
 import cx from 'classnames';
+import { props, t } from 'tcomb-react';
 
 export skinnable, { contains } from 'react-skinnable';
-export { props, t } from 'tcomb-react';
+export { props, t };
 export const stateClassUtil = (...classes) => cx([].concat(...classes).map(cl => `is-${cl}`));
+
+export const getContextWrapper = (contextTypes) => {
```

Comment:
minor, but the same exact thing I think is already implemented for Modal here: https://github.com/buildo/react-components/blob/master/src/modal/modalWithContext.js
What about reusing/sharing?